README.md

```diff
@@ -1,25 +1,22 @@
-# CSDE-cordapp-template-kotlin
+# release-V5 cordapp-template-kotlin
```

Comment:
previously this was the name of the repository; now it is a specific branch name where the code would live, and the target repo name